docs/plugins/discovery-gce.asciidoc

```diff
@@ -26,12 +26,14 @@ The following gce settings (prefixed with `cloud.gce`) are supported:
 
  `project_id`::
 
-     Your Google project id (mandatory).
+     Your Google project id.
+     If the value is not specified, default project id is picked up from a metadata server.
```

Comment:
Can we add a note that the GOOGLE env var are not picked up?

Reply:
++